tests/mock_network/mod.rs

```diff
@@ -396,11 +396,7 @@ fn sibling_knowledge_update_after_split() {
                 return false;
             }
 
-            if !node
-                .inner
-                .get_their_knowledge()
-                .contains_key(&node.our_prefix().sibling())
-            {
+            if node.inner.get_their_knowledge(&node.our_prefix().sibling()) == 0 {
```

Comment:
== 0 ?

Reply:
`get_their_knowledge` returns the index in **our** `SectionProofChain` of the key **they** know. If it returns 0, that means they know* only the root/genesis key which every section implicitly knows, so in other words, they don't know us yet.

*) more precisely: "we know that they know"